src/web/basics/button/tests/tests-button.html

```diff
@@ -10,11 +10,12 @@
         </style>
     </head>
     <body>
-
-        <div class='regular-button buttonHolder'></div>
-        <div class='dispose-button buttonHolder'></div>
-        <div class='updated-button buttonHolder'></div>
-        <div class='destroy-button buttonHolder'></div>
+        <div class='test-container' style="padding: 40px; margin: 0;">
```

Comment:
The reason that we did this, is give more padding around the actually buttons for the snapshot. Otherwise if we added padding to body, that padding did not get captured in the snapshot, and it appeared  that the buttons would be slightly cut off along the border. See the example below. The actual view is with padding on the body instead of the test container class-

![image](https://cloud.githubusercontent.com/assets/5723109/26286949/69321a4e-3e25-11e7-9b15-80cccd2e6feb.png)